requirements.txt

```diff
@@ -0,0 +1,8 @@
+cycler==0.10.0
```

Comment:
These are all of the dependencies of the project. The "root" dependencies (the ones the project actually imports) are `matplotlib` and `numpy`. The other dependencies, like `cycler` and `kiwisolver`, are required by `matplotlib` and `numpy` so we must also install them. The reason to include them in this file is to lock the version that is installed so behavior is consistent between users.

Reply:
This file was generated by

```bash
pip freeze > requirements.txt
```